_includes/head.html

```diff
@@ -24,6 +24,7 @@
   <link rel="manifest" href="assets/img/favicon/site.webmanifest">
   <link rel="mask-icon" href="assets/img/favicon/safari-pinned-tab.svg" color="#08a0c4">
   <link rel="shortcut icon" href="assets/img/favicon/favicon.ico">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
```

Comment:
Remove this and make it another pull request. One that also changes all the icons you can find.